keyboards/westfoxtrot/aanzee/keymaps/via/rules.mk

```diff
@@ -0,0 +1,70 @@
+# MCU name
```

Comment:
You should only need the VIA specific stuff here. The rest of this is redundant. 

Eg, you should only need:
```make
#add files needed for via

 RAW_ENABLE = yes
DYNAMIC_KEYMAP_ENABLE = yes
SRC += keyboards/wilba_tech/wt_main.c
```

Reply:
That's not what I am doing, for $reasons. Leave it alone.

Reply:
@Wilba6582 Could you clarify what you mean here?  

This is a "via" keymap, and as such should only need the via specific lines.  It should inherit everything else. 
So, we generally don't want the "entire" rules.mk content here, which is what my comment was referring to.

However, if you don't want the VIA stuff here, at all, then that's fine.

But it's not clear to me what you mean.

Reply:
The `rules.mk` in keymaps trumps the keyboard `rules.mk`. To make absolutely sure enabling features don't break VIA support, I like how a lot of these are explicitly defined in keymaps. However in the future, we can probably refrain from copy and pasting the whole keyboard `rules.mk` as things like mcu should never ever be changed. 

Merging this in NOW!

Reply:
It's a placeholder for me coming back to these and refactoring it down to the things that need to be explicitly set for VIA compatibility (esp. feature defines), not that it wouldn't work now as just the delta with the parent.